### ISSUE TITLE
Translation from localtime to GMT(DST aware) is too slow

### ISSUE THREAD

**Issue by shino** [webmachine/webmachine#157]
erlang:localtime_to_universaltime/2, slow if 2nd arg is true. It takes a few tens of msec when executed by multi processes.
https://gist.github.com/shino/d0fb2a904955f66420b8

(`bmp/3` is a utility function in my `user_default`: https://gist.github.com/shino/5963764)

This function is used internally at last-modifed header formatting:

```
                    wrcall({set_resp_header, "Last-Modified",
                           httpd_util:rfc1123_date(
                             calendar:universal_time_to_local_time(LM))})
```

Because `LM` is already GMT, round trip GMT->Local->GMT is not necessary.

**Comment by shino** [comment]
Addressed by PR #158

**Comment by shino** [comment]
#158 merged.
